module/move/wca/src/ca/executor/converter.rs

```diff
@@ -19,7 +19,7 @@ pub( crate ) mod private
   ///
   /// ```
   /// # use wca::{ Command, Type, GrammarCommand, ExecutorConverter, Routine };
-  /// # use wtools::HashMap;
```

Comment:
because with `wtools::HashMap` doc tests crashed, and even in the documentation for `wtools::HashMap` in the examples std::collections::Hashmap is [used](https://docs.rs/wtools/latest/wtools/struct.HashMap.html#:~:text=Examples-,use%20std%3A%3Acollections%3A%3AHashMap%3B,-//%20Type%20inference%20lets).